content/docs/get-started/serving.md

```diff
@@ -1,81 +1,40 @@
 # Serving models
 
-For online serving, you can create a server from your model. We will try out
-FastAPI server. All available server implementations are listed
-[here](/doc/user-guide/mlem-abcs#server).
+If you want to serve your model online, MLEM can do that for you using FastAPI
+or, for example, RabbitMQ. We'll check out how it works with FastAPI since
+serving models via REST API is quite common.
 
 ## Running server
 
-To start up FastAPI server run:
+To start up a FastAPI server run:
 
 ```cli
-$ mlem serve rf fastapi
-⏳️ Loading model from .mlem/model/rf.mlem
+$ mlem serve fastapi --model models/rf
+⏳️ Loading model from models/rf.mlem
 Starting fastapi server...
-🖇️ Adding route for /predict
-🖇️ Adding route for /predict_proba
-🖇️ Adding route for /sklearn_predict
-🖇️ Adding route for /sklearn_predict_proba
+🖇️  Adding route for /predict
+🖇️  Adding route for /predict_proba
+🖇️  Adding route for /sklearn_predict
+🖇️  Adding route for /sklearn_predict_proba
 Checkout openapi docs at <http://0.0.0.0:8080/docs>
-INFO:     Started server process [2917]
+INFO:     Started server process [16696]
 INFO:     Waiting for application startup.
 INFO:     Application startup complete.
 INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
 ```
 
-Servers automatically create endpoints from model methods with payload schemas
-corresponding to serialized dataset types.
+Servers automatically create endpoints from model methods using `sample_data`
+argument provided to [mlem.api.save](/doc/api-reference/save).
+
+That's it! You can check out the other available server implementations in
+[User Guide](/doc/user-guide/serving/).
 
 ## Making requests
 
-You can open Swagger UI (OpenAPI) at
+While serving the model, you can open Swagger UI (OpenAPI) at
 [http://localhost:8080/docs](http://localhost:8080/docs) to check out OpenAPI
```

Comment:
_From https://github.com/iterative/mlem.ai/pull/191#issuecomment-1294488343_:

> localhost link should be in code. Idea is that right before that link you run mlem serve, that spins up a server there
- @aguschin 

Oh I see. Well, I don't think we need to make it an actual link, maybe just a code block you can copy easily?

> If there is a way to disable the check for that specific link, please share that

I don't think so but you can check with @iterative/websites

Reply:
If it's for link checker CI, we can add the following line to https://github.com/iterative/mlem.ai/blob/main/config/link-check/excluded-links.yml
```
- 'http://localhost:8000**'
```

Reply:
Thanks @yathomasi, please see this PR https://github.com/iterative/mlem.ai/pull/216